cdap-ui/app/cdap/components/FastAction/SetPreferenceAction/SetPreferenceModal.js

```diff
@@ -469,9 +472,7 @@ export default class SetPreferenceModal extends Component {
                 ) : null}
               </div>
               <div className="preferences-error">
-                {this.state.error ? (
-                  <div className="bg-danger text-white">{this.state.error}</div>
-                ) : null}
+                {this.state.error ? <div className="text-danger">{this.state.error}</div> : null}
```

Comment:
Could we use `If` component instead of ternary operator?

Reply:
Done!